src/garage/np/_functions.py

```diff
@@ -1,7 +1,67 @@
 """Utility functions for NumPy-based Reinforcement learning algorithms."""
 import numpy as np
 
+from garage._dtypes import TrajectoryBatch
 from garage.misc import tensor_utils
+from garage.sampler.utils import rollout
+
+
+def process_paths(paths):
```

Comment:
i still think this is an excessively-vague name to be in `garage.np` -- if you tab-completed it, you wouldn't know what it does.

How about `samples_to_tensors` which is what it appears to be doing.

Reply:
Sure, fixed.